examples/basic.rs

```diff
@@ -19,10 +16,6 @@ fn main() {
         .add_plugin(FrameTimeDiagnosticsPlugin::default())
         .add_plugin(LogDiagnosticsPlugin::default())
         .add_plugins(DefaultPlugins.set(WindowPlugin {
-            window: WindowDescriptor {
-                present_mode: PresentMode::AutoNoVsync,
-                ..Default::default()
-            },
             ..Default::default()
```

Comment:
If we are removing the options here, we can just make this `.add_plugins(DefaultPlugins)`

Reply:
ah, yes, I kind of just removed it because I wanted to make it work first, then perhaps find out what the new api is.

however, I think it probably makes sense not to have this stuff in the basic example. probably better to have it clean and minimal, and create another example for manual benchmarking?

Reply:
Probably belongs in another PR, though. I'll just bring back the original behavior, to keep this one focused/uncontroversial.